src/TestCodeFiles/RuntimeSystemTests/Test1/Test1Driver.java

```diff
@@ -0,0 +1,16 @@
+package TestCodeFiles.RuntimeSystem.Test1;
+
+//In runTime test1 we assume all the input commands have correct syntax
+
+public class Test2Driver {
+	public static void main(String[] args) {
+		
+		String st = "add 78 to 9 \n" + "                      \n" + "                      \n"
```

Comment:
While the input for the test is good, plz stick to the input we decided on in the documentation.